ci/build_common.sh

```
@@ -91,11 +95,15 @@ BUILD_DIR="../build/${CCCL_BUILD_INFIX}"
 mkdir -p $BUILD_DIR
-rm -f ../build/latest
-ln -sf $BUILD_DIR ../build/latest
-
-# Now that BUILD_DIR exists, use readlink to canonicalize the path:
-BUILD_DIR=$(readlink -f "${BUILD_DIR}")
+if [[ "${HOST_OS}" == "windows" ]]; then
+  # Git Bash on Windows cannot create directory symlinks without elevated privileges
+  BUILD_DIR=$(cd "${BUILD_DIR}" && pwd)
+else
+  rm -f ../build/latest
+  ln -sf $BUILD_DIR ../build/latest
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)